app/src/androidTest/java/net/skyscanner/backpack/compose/dialog/BpkDialogTest.kt

```diff
@@ -115,6 +117,24 @@ class BpkDialogTest : BpkSnapshotTest() {
     }
   }
 
+  @Test
+  fun imageStartAlignment() {
+    setDimensions(height = 700, width = 420)
+    assumeVariant(BpkTestVariant.Default, BpkTestVariant.DarkMode, BpkTestVariant.Rtl)
```

Comment:
this shouldn't be needed - the `record` function already disables the themed variant

Reply:
CI and local snapshot recording fails without this line. It works on other components but not this one

Reply:
huh very odd - what's the failure?

Reply:
looking at CI there was 1 failure on one of the variants and that was an emulator timeout, so I think it can be safely removed

Reply:
it gets blocked on a white screen and times out after a minute

Reply:
I see - my guess is it was a one off as both of the checks to exactly the same. Could you try it again locally and see if it times out? Otherwise fine to keep it